exercises/robot-simulator/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+#[derive(PartialEq, Debug)]
+pub enum Direction {
+    North,
+    East,
+    South,
+    West,
+}
+
+pub struct Robot {
+}
+
+impl Robot {
+    pub fn new(x: isize, y: isize, d: &Direction) -> Self {
+        Robot {}
+    }
+
+    pub fn turn_right(&mut self) {}
```

Comment:
I would pre-fill the functions with `unimplemented!()` just to make sure, that you know where to put the implementation. This is more important for functions which return values, because `unimplemented!()` always typechecks, no matter the type.